tools/sgtrivy/tools.go

```diff
@@ -0,0 +1,99 @@
+package sgtrivy
+
+import (
+	"context"
+	_ "embed"
+	"errors"
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"runtime"
+
+	"go.einride.tech/sage/sg"
+	"go.einride.tech/sage/sgtool"
+)
+
+//go:embed trivyignore
+var DefaultConfig []byte
```

Comment:
IMO hide this until someone really needs it, in case we want to change the public API?

Reply:
Good point. Copy/Paste from another package